README.md

```diff
@@ -7,7 +7,7 @@
 -	Calculating the area of rectangle using different method program 
```

Comment:
Some points starts with Capital letter some in small letter, please make sure it is same all the place

Reply:
done